src/lib/import-export/import/importers/importer.ts

```diff
@@ -49,6 +50,40 @@ export class DefaultImporter extends EventEmitter implements Importer {
 		}
 	}
 
+	protected async hasCreateWithoutDrop( sqlFiles: string[] ): Promise< boolean > {
```

Comment:
What would have happened if we simplified this and removed the database each time?

Reply:
Yep, I think we could always reset the database when importing. My only concern would be if we allow partial importation of a database, but I understand this is not the case.

Reply:
the problem was that this import flow was executed by two different file sources: Jetpack backups and single SQL files.
Now I've refactored it to have two different importers and for Jetpack backups remove the database always. https://github.com/Automattic/studio/pull/424/commits/27ce87f037a637b009f974d295715119ccca9d60